<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterWidget</class>
 <widget class="QWidget" name="ParameterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParameterWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <widget class="QFrame" name="ResultsFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>246</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Results</string>
      </property>
     </widget>
     <widget class="QLabel" name="paramLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="passLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Worst Margin</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="title">
             <string>Pair</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QListWidget" name="marginListWidget">
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
              </widget>
             </item>
            </layout>
            <zorder>marginListWidget</zorder>
            <zorder></zorder>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="verticalSpacing">
           <number>12</number>
          </property>
          <item row="0" column="1">
           <widget class="QLabel" name="marginValueLabel">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Freq : </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="marginFreqLabel">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Limit : </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="marginLimitLabel">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Margin : </string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="marginLabel">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Value :</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder>horizontalLayoutWidget_2</zorder>
       <zorder></zorder>
       <zorder></zorder>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Worst Value</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="topMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="title">
             <string>Pair</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QListWidget" name="worstListWidget">
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <property name="formAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="verticalSpacing">
             <number>12</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Value :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="worstValueLabel">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Freq : </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="worstFreqLabel">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Limit : </string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="worstLimitLabel">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Margin : </string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="worstMarginLabel">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
